drivers/gles2/rasterizer_storage_gles2.cpp

```diff
@@ -5783,27 +5783,29 @@ void RasterizerStorageGLES2::initialize() {
 
 	config.keep_original_textures = false;
 	config.shrink_textures_x2 = false;
+	config.depth_internalformat = GL_DEPTH_COMPONENT;
+	config.depth_type = GL_UNSIGNED_INT;
 
 #ifdef GLES_OVER_GL
 	config.float_texture_supported = true;
 	config.s3tc_supported = true;
 	config.pvrtc_supported = false;
 	config.etc1_supported = false;
 	config.support_npot_repeat_mipmap = true;
-	config.depth_internalformat = GL_DEPTH_COMPONENT;
-	config.depth_type = GL_UNSIGNED_INT;
+	config.depth_buffer_internalformat = GL_DEPTH_COMPONENT24;
```

Comment:
I've tried looking whether it's OK to use `GL_DEPTH_COMPONENT24` for `glRenderbufferStorage` on OpenGL 2.1, but navigating the specs and various extensions is as painful as ever...

What I found so far:
- `glRenderbufferStorage` is not core in OpenGL 2.1, we use `glRenderbufferStorageEXT` from https://www.khronos.org/registry/OpenGL/extensions/EXT/EXT_framebuffer_object.txt, which only defines `STENCIL_INDEX{1,4,8,16}_EXT` as valid `internalFormat` values.
Amazingly the examples in `EXT_framebuffer_object` do use `GL_DEPTH_COMPONENT24` but don't specify where they take it from.
- `glTexImage2D` for OpenGL 2.1 mentions that `GL_DEPTH_COMPONENT24` is available with GL 1.4+. It's however in the content of `glTexImage2D`, it doesn't tell us if it's available for `EXT_framebuffer_object`.
- https://www.khronos.org/registry/OpenGL/extensions/OES/OES_depth24.txt adds `GL_DEPTH_COMPONENT24` for the OpenGL **ES** extension. As I understand it there `glRenderbufferStorage` is core in ES 2.0, but this extension is still needed for the 24-bit component.

What a mess... I guess if it compiles, it's fine...